packages/expo-secure-store/src/SecureStore.ts

```diff
@@ -66,6 +66,16 @@ export type SecureStoreOptions = {
    * > If the item is set with the `keychainService` option, it will be required to later fetch the value.
    */
   keychainService?: string;
+  /**
+   * - iOS: Equivalent to `kSecAccessControlUserPresence`
+   * - Android: Equivalent to `setUserAuthenticationRequired(true)` (requires API 23). Needs to generate
+   * a new encryption key to fully work, it is recommended to use it with separate `keychainService`
```

Comment:
```suggestion
   * - iOS: Equivalent to `kSecAccessControlUserPresence`
   * - Android: Equivalent to `setUserAuthenticationRequired(true)` (requires API 23). Complete functionality 
   *  is unlocked only with a freshly generated key; it is recommended to use this mechanism with a separate `keychainService` <separate from what?>.
```
I guess all this explanation is only about Android part right? Additionally this message is targeted to the devs that have already used this mechanism in theirs apps? 🤔 

I would go with something like this:
```suggestion
   * Option responsible for enabling the usage user authentication methods available on the device while 
   * accessing data saved in SecureStore.
   *
   * - iOS: Equivalent to `kSecAccessControlUserPresence`
   * - Android: Equivalent to `setUserAuthenticationRequired(true)` (requires API 23). Complete functionality 
   * is unlocked only with a freshly generated key - this would not work in tandem with the `keychainService` 
   * value used for the others non-authenticated operations.
```

Reply:
Yes, it's only about Android but not necessarily for the devs that have already used it. Your suggestion it pretty much spot on and I thought it was important to mention that but I could've worded it much better.